### PR TITLE
Create IT.yaml

### PR DESCRIPTION
**Pull request by Bu22o**
hashid: fg9cR

### CODE REVIEW

File: IT.yaml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+home.nick.title.keep: Full Stack Intern
+home.paddy.desc.keep: >-
+ Senior Software Engineer presso Microsoft, specializzato in sistemi distribuiti.
+  Contributore attivo in progetti open source. M.S. Computer Science presso Yale
+  Università.
+home.paddy.name.keep: Paddy Kan
+home.paddy.title.keep: Distributed Systems
+home.powered-by: Alimentato da una blockchain incentrata sulla privacy
+home.qevan.desc.keep: >-
+  Ph.D in Machine Learning and Computer Vision at National University of
```
Comment (WildLifeblood) [inline IT.yaml]: 'and' and 'at' should be translated

File: IT.yaml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  Singapore. Engineering Manager and Research Scientist at Facebook. Growth
```
Comment (WildLifeblood) [inline IT.yaml]: I think you forgot translating this line.

File: IT.yaml [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  Hacker. Startup investor and advisor. 20+ publications / patents.
```
Comment (WildLifeblood) [inline IT.yaml]: And this too

File: IT.yaml [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+home.qevan.name.keep: Qevan Guo
+home.qevan.title.keep: Co-founder
+home.raullen.desc.keep: >-
+  Ph.D in Machine Learning e Computer Vision presso la National University of
+  Singapore. Responsabile tecnico e ricercatore scientifico su Facebook. Growth hacker. Investitore e consulente alle startup. più di 20 pubblicazioni / brevetti.
```
Comment (WildLifeblood) [inline IT.yaml]: If you can translate 'Growth hacker' do it.

File: IT.yaml [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+home.raullen.name.keep: Raullen Chai
+home.raullen.title.keep: Co-founder
+home.section.advisors: IoTeX.Advisors
```
Comment (WildLifeblood) [inline IT.yaml]: 'Advisors' needs to be translated

File: IT.yaml [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+home.section.arch: IoTeX.Architettura
+home.section.innovations: IoTeX.Innovazioni
+home.section.investors: IoTeX.Investitori
+home.section.partners: IoTeX.PartnerStrategici
+home.section.roadmap: IoTeX.Roadmap
```
Comment (WildLifeblood) [inline IT.yaml]: From 153 to 156 it is untranslated.

File: IT.yaml [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+home.vedaad.desc.keep: >-
+ Studente presso UC Berkeley, studia EECS e Matematica applicata; ricercatore di blockchain
+  nel team criptoeconomico di Blockchain @ Berkeley
+home.vedaad.name.keep: Vedaad Shakib
+home.vedaad.title.keep: Blockchain Engineering Intern
+home.vincent.desc.keep: >-
+ Laurea in Informatica presso l'Università di Waterloo.Appassionato di Sicurezza e blockchain.
+  
+home.vincent.name.keep: Vincent Vuong
+home.vincent.title.keep: Software Engineer Intern
```
Comment (WildLifeblood) [inline IT.yaml]: I would translate 'Intern'

File: IT.yaml [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+subscribe.desc: Con i nostri ultimi progressi ed eventi.
+subscribe.title: RESTA AGGIORNATO
+testnet-preview.home-title: L'anteprima della Testnet(StoneVan) sarà rilasciata il
+testnet-preview.released-event-title: La nostra anteprima della Testnet è stata rilasciata!
+testnet-preview.released-title: L'anteprima della Testnet è stata rilasciata
+topbar.blog: BLOG
+topbar.careers: CARRIERE
+topbar.charity: CARITÀ
+topbar.faq: FAQ
+topbar.home: HOME
```
Comment (WildLifeblood) [inline IT.yaml]: You forgot this :)

File: IT.yaml [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+home.site.description: >-
+  Una rete decentralizzata per Internet of Things (IoT), alimentata da una
+  Blockchain auto-scalabile, estensibile, incentrata sulla privacy. Collegamento del mondo fisico
+  blocco dopo blocco
+home.slogan: Collegamento del mondo fisico Blocco dopo blocco
+home.vedaad.desc.keep: >-
+ Studente presso UC Berkeley, studia EECS e Matematica applicata; ricercatore di blockchain
+  nel team criptoeconomico di Blockchain @ Berkeley
+home.vedaad.name.keep: Vedaad Shakib
+home.vedaad.title.keep: Blockchain Engineering Intern
```
Comment (WildLifeblood) [inline IT.yaml]: 'Intern' should be translated.

File: IT.yaml [hunk 10]
```diff
@@ -0,0 +1,218 @@
...
+Master in Computer Science presso USC e Bachelor presso University of Wisconsin-
+  Madison. Appassionato di sviluppo web, blockchain e machine learning.
+home.nick.name.keep: Nick Chen
+home.nick.title.keep: Full Stack Intern
+home.paddy.desc.keep: >-
+ Senior Software Engineer presso Microsoft, specializzato in sistemi distribuiti.
+  Contributore attivo in progetti open source. M.S. Computer Science presso Yale
+  Università.
+home.paddy.name.keep: Paddy Kan
+home.paddy.title.keep: Distributed Systems
```
Comment (WildLifeblood) [inline IT.yaml]: It has to be translated

File: IT.yaml [hunk 11]
```diff
@@ -0,0 +1,218 @@
...
+  numerose aziende. Vasta esperienza in startup, investimenti e costruzione
+  di ecosistemi.
+home.jing.name.keep: Jing Sun
+home.jing.title.keep: Co-founder
+home.larry.desc.keep: >-
+  Consulente di gestione presso Oliver Wyman, specializzato in operazioni digitali. doppio in
+  B.S. in Finance & Economics, minore in Visual Arts & Design del MIT.
+  Blockchain e appassionato di IoT.
+home.larry.name.keep: Larry Pang
+home.larry.title.keep: Growth & Community
```
Comment (WildLifeblood) [inline IT.yaml]: It has to be translated.

File: IT.yaml [hunk 12]
```diff
@@ -0,0 +1,218 @@
...
+home.fan.title.keep: null
+home.guang.desc.keep: >-
+Membro del IEEE. Professore all'Università di Waterloo, Canada. Top Scientist in
+  crittografia e crittografia leggera, reti ad hoc,radiofrequenza e
+  Identificazione (RFID) di sicurezza.
+home.guang.name.keep: Guang Gong
+home.guang.title.keep: null
+home.jack.desc.keep: >-
+  Co-Founding Managing Partner di HCM Capital, Foxconn Technology Group.
+  Co-fondatore e direttore esecutivo di Chained Finance Ltd.,Direttore fondatore ed esecutivo
```
Comment (WildLifeblood) [inline IT.yaml]: Care with spaces.

File: IT.yaml [hunk 13]
```diff
@@ -0,0 +1,218 @@
...
+IoTeX sta affrontando questo problema introducendo la token economy negli spazi IoT come
+  crediamo fortemente che gli incentivi crittografici e lo sforzo della comunità siano
+  le due forze chiave per spingere le innovazioni nello spazio IoT. In breve, IoTeX è la
+  piattaforma blockchain orientata all'IoT di nuova generazione con forte scalabilità,
+  privacy, isolabilità e sviluppo per l'incubazione di nuove applicazioni IoT
+  e ecosistemi.
+home.decentralized-protocol: Una rete decentralizzata per L'Internet of Things
+home.dustin.desc.keep: >-
+  Ph.D in Data Security presso USC. Software engineer dello staff di Intel. 12 anni di
+  esperienza nei sistemi distribuiti, protezione dei contenuti,accelerazione video hardware
```
Comment (WildLifeblood) [inline IT.yaml]: Care with spaces before 'accelerazione'

File: IT.yaml [hunk 14]
```diff
@@ -0,0 +1,218 @@
...
+  Apprezziamo il tuo grande cuore! Oltre ai 16.000 punti unici
+  condivideremo la tua buona azione sul sito di IoTeX!
+charity.tweet.summary.image: >-
+  https://s3.amazonaws.com/web-iotex-static/home/1st-charity-twitter-summary-image-v2.png
+charity.tweet.summary.title: Sei sulla IoTeX Honor Board!
+claim.btn: Reclama i tuoi punti
+contact.btn: Contattaci
+contact.desc: Gli esperti di IoTeX sono pronti a rispondere alle tue domande tramite support@iotex.io
+contact.title: Rimani in contatto
+ethFoundation: Ethereum Foundation
```
Comment (WildLifeblood) [inline IT.yaml]: I would translate 'Foundation'

File: IT.yaml [hunk 15]
```diff
@@ -0,0 +1,218 @@
...
+hall-of-fame.badge-section.title: IoTeX 1st Programma di beneficenza— ETH Foundation
```
Comment (WildLifeblood) [inline IT.yaml]: 'Foundation' here too.

File: IT.yaml [hunk 16]
```diff
@@ -0,0 +1,218 @@
...
+ Ethereum ha rivoluzionato il mondo blockchain negli ultimi quattro anni è
+  fatto della parola "blockchain" la parola piu ricercata in tempi odierni. Come leader del settore, Ethereum ha
+  fornito <a target = "_ blank"
+  href = ""> https://blog.ethereum.org/2018/03/07/announcing-beneficiaries-ethereum-foundation-grants/ sovvenzioni </a>
+  per alimentare ricercatori e sviluppatori a lavorare su vari fronti per supportare
+  l'intero ecosistema blockchain. IoTeX crede fortemente che possiamo muoverci
+  insieme lungo questo settore grazie a ricerche e collaborazioni rigorose.
+  Abbiamo bisogno di talenti e di menti brillanti per fare colpo sul mondo. Un piccolo
+  ma efficace modo per aiutare la nostra causa è donare ETH all'Ethereum
+  Foundation per aiutare a finanziare la ricerca blockchain.
```
Comment (WildLifeblood) [inline IT.yaml]: 'Foundation' here too.

File: IT.yaml [hunk 17]
```diff
@@ -0,0 +1,218 @@
...
+ Il crowdsourcing come modo di finanziare una causa specifica per un ente di beneficenza non è nuovo, ma
+  IoTeX lo sta facendo tramite "contratti intelligenti" (Smart Contract) e utilizzando l'hype generato dalla criptovaluta per cause caritatevoli. Nei prossimi mesi, sotto il suo programma di premiazione comunitaria, IoTeX continuerà a selezionare alcuni programmi di beneficenza che si allineano con
+  La missione di IoTeX di "Connettere il mondo fisico, blocco dopo blocco" come parte del
+  Programma di ricompensa della comunità di IoTeX.
+
+
+  IoTeX crede nella crescita insieme alla comunità e nella connessione di
+  mondo fisico blocco per blocco. È un obiettivo molto ambizioso ma può essere
+  raggiunto con una squadra forte e una community solidale. Insieme alla
+  community per spingere la frontiera per lo sviluppo della blockchain
```
Comment (WildLifeblood) [inline IT.yaml]: 'community' has to be translated.

File: IT.yaml [hunk 18]
```diff
@@ -0,0 +1,218 @@
...
+home.dustin.name.keep: Dustin Xie
+home.dustin.title.keep: Blockchain Engineer
+home.fan.desc.keep: >-
+in precedenza è stato socio fondatore di Sequoia Capital China e
+  Direttore di DFJ ePlanet Ventures. Classificato da Forbes come uno dei principali venture
+  capitalist nella lista annuale di Midas in Cina.
+home.fan.name.keep: Fan Zhang
+home.fan.title.keep: null
+home.guang.desc.keep: >-
+Membro del IEEE. Professore all'Università di Waterloo, Canada. Top Scientist in
```
Comment (WildLifeblood) [inline IT.yaml]: I would translate 'Top Scientist', it isn't English fixed.

File: IT.yaml [hunk 19]
```diff
@@ -0,0 +1,218 @@
...
+  crittografia e crittografia leggera, reti ad hoc,radiofrequenza e
+  Identificazione (RFID) di sicurezza.
+home.guang.name.keep: Guang Gong
+home.guang.title.keep: null
+home.jack.desc.keep: >-
+  Co-Founding Managing Partner di HCM Capital, Foxconn Technology Group.
```
Comment (WildLifeblood) [inline IT.yaml]: I would translate 'Co-Founding Managing Partner'

File: IT.yaml [hunk 20]
```diff
@@ -0,0 +1,218 @@
...
+  Co-fondatore e direttore esecutivo di Chained Finance Ltd.,Direttore fondatore ed esecutivo
+   di FnConn e Board Director di Abra.
```
Comment (WildLifeblood) [inline IT.yaml]: I would translate 'Board Director'

File: IT.yaml [hunk 21]
```diff
@@ -0,0 +1,218 @@
...
+home.jack.name.keep: Jack Lee
+home.jack.title.keep: null
+home.jing.desc.keep: >-
+ Socio fondatore di Sparkland Capital, che si concentra su Frontier Tech, guida più di 40
+  investimenti e innovazioni in blockchain, sicurezza e IoT. Consigliere di amministrazione in
+  numerose aziende. Vasta esperienza in startup, investimenti e costruzione
+  di ecosistemi.
+home.jing.name.keep: Jing Sun
+home.jing.title.keep: Co-founder
```
Comment (WildLifeblood) [inline IT.yaml]: You forgot to translate this.

File: IT.yaml [hunk 22]
```diff
@@ -0,0 +1,218 @@
...
+  Consulente di gestione presso Oliver Wyman, specializzato in operazioni digitali. doppio in
+  B.S. in Finance & Economics, minore in Visual Arts & Design del MIT.
+  Blockchain e appassionato di IoT.
+home.larry.name.keep: Larry Pang
+home.larry.title.keep: Growth & Community
+home.monika.desc.keep: >-
+  Specialista in tecnologie di comunicazione con profonda competenza in materia di sicurezza informatica,
+  privacy, networking e tecnologie emergenti.
+home.monika.name.keep: Monika Hathaway
+home.monika.title.keep: Communication
```
Comment (WildLifeblood) [inline IT.yaml]: It has to be translated.

File: IT.yaml [hunk 23]
```diff
@@ -0,0 +1,218 @@
...
+home.nick.desc.keep: >-
+Master in Computer Science presso USC e Bachelor presso University of Wisconsin-
+  Madison. Appassionato di sviluppo web, blockchain e machine learning.
+home.nick.name.keep: Nick Chen
+home.nick.title.keep: Full Stack Intern
```
Comment (WildLifeblood) [inline IT.yaml]: 'Intern' can be translated

File: IT.yaml [hunk 24]
```diff
@@ -0,0 +1,218 @@
...
+  Università.
+home.paddy.name.keep: Paddy Kan
+home.paddy.title.keep: Distributed Systems
+home.powered-by: Alimentato da una blockchain incentrata sulla privacy
+home.qevan.desc.keep: >-
+  Ph.D in Machine Learning and Computer Vision at National University of
+  Singapore. Engineering Manager and Research Scientist at Facebook. Growth
+  Hacker. Startup investor and advisor. 20+ publications / patents.
+home.qevan.name.keep: Qevan Guo
+home.qevan.title.keep: Co-founder
```
Comment (WildLifeblood) [inline IT.yaml]: It has to be translated.

File: IT.yaml [hunk 25]
```diff
@@ -0,0 +1,218 @@
...
+home.raullen.desc.keep: >-
+  Ph.D in Machine Learning e Computer Vision presso la National University of
+  Singapore. Responsabile tecnico e ricercatore scientifico su Facebook. Growth hacker. Investitore e consulente alle startup. più di 20 pubblicazioni / brevetti.
+home.raullen.name.keep: Raullen Chai
+home.raullen.title.keep: Co-founder
```
Comment (WildLifeblood) [inline IT.yaml]: It has to be translated.

File: IT.yaml [hunk 26]
```diff
@@ -0,0 +1,218 @@
...
+ Laurea in Informatica presso l'Università di Waterloo.Appassionato di Sicurezza e blockchain.
+  
+home.vincent.name.keep: Vincent Vuong
+home.vincent.title.keep: Software Engineer Intern
+home.xinxin.desc.keep: >-
+ Ph.D in Cryptography presso UWaterloo. Sr. Research Engineer presso Bosch Research. 14
+  anni di ricerca e esperienza nel settore della sicurezza delle informazioni e della crittografia.
+  40 e più pubblicazioni / brevetti.
+home.xinxin.name.keep: Xinxin Fan
+home.xinxin.title.keep: Head of Cryptography
```
Comment (WildLifeblood) [inline IT.yaml]: It has to be translated.

File: IT.yaml [hunk 27]
```diff
@@ -0,0 +1,218 @@
...
+  IoTeX lo sta facendo tramite "contratti intelligenti" (Smart Contract) e utilizzando l'hype generato dalla criptovaluta per cause caritatevoli. Nei prossimi mesi, sotto il suo programma di premiazione comunitaria, IoTeX continuerà a selezionare alcuni programmi di beneficenza che si allineano con
+  La missione di IoTeX di "Connettere il mondo fisico, blocco dopo blocco" come parte del
+  Programma di ricompensa della comunità di IoTeX.
+
+
+  IoTeX crede nella crescita insieme alla comunità e nella connessione di
+  mondo fisico blocco per blocco. È un obiettivo molto ambizioso ma può essere
+  raggiunto con una squadra forte e una community solidale. Insieme alla
+  community per spingere la frontiera per lo sviluppo della blockchain
+  ecosistema, IoTeX spera davvero che la community possa capire l'intenzione originale
```
Comment (WildLifeblood) [inline IT.yaml]: Again, 'community' has to be translated.

File: IT.yaml [hunk 28]
```diff
@@ -0,0 +1,218 @@
...
+  privacy, isolabilità e sviluppo per l'incubazione di nuove applicazioni IoT
+  e ecosistemi.
+home.decentralized-protocol: Una rete decentralizzata per L'Internet of Things
+home.dustin.desc.keep: >-
+  Ph.D in Data Security presso USC. Software engineer dello staff di Intel. 12 anni di
+  esperienza nei sistemi distribuiti, protezione dei contenuti,accelerazione video hardware
+  , streaming video di rete, driver grafici. Crittografia e
+  appassionato di blockchain.
+home.dustin.name.keep: Dustin Xie
+home.dustin.title.keep: Blockchain Engineer
```
Comment (WildLifeblood) [inline IT.yaml]: You can translate 'Engineer'

**Comment by loriIoTeX** [comment]
Thank you for helping us with the translation! After carefully review the contents, unfortunately we decided to close this pull request. However, bonus points for the high-quality translations have been distributed to your Telegram account, please check with our bounty bot. We appreciate your continuous support in the future!

**Pull request closed without merging**